src/transformers/generation/streamers.py

```diff
@@ -127,6 +131,30 @@ def on_finalized_text(self, text: str, stream_end: bool = False):
         """Prints the new text to stdout. If the stream is ending, also prints a newline."""
         print(text, flush=True, end="" if not stream_end else None)
 
+    def _is_chinese_char(self, cp):
+        """Checks whether CP is the codepoint of a CJK character."""
+        # This defines a "chinese character" as anything in the CJK Unicode block:
+        #   https://en.wikipedia.org/wiki/CJK_Unified_Ideographs_(Unicode_block)
+        #
+        # Note that the CJK Unicode block is NOT all Japanese and Korean characters,
+        # despite its name. The modern Korean Hangul alphabet is a different block,
+        # as is Japanese Hiragana and Katakana. Those alphabets are used to write
+        # space-separated words, so they are not treated specially and handled
+        # like the all of the other languages.
+        if (
+            (cp >= 0x4E00 and cp <= 0x9FFF)
+            or (cp >= 0x3400 and cp <= 0x4DBF)  #
```

Comment:
Please remove the trailing `#` character :)

(here and on the subsequent lines)

Reply:
I'm not quite sure, I just copy is from https://github.com/huggingface/transformers/blob/fb3aa06cb673d0e2774a2924747d3290135c09cc/src/transformers/models/bert/tokenization_bert.py#L481-L503, and it looks like that all the `_is_chinese_char` functions in the repo is the same.

Reply:
hah, I didn't know of this previous code block! 

In that case yes, let's leave the two equal 👍